Dockerfile

```diff
@@ -21,12 +21,17 @@ ENV         PATH="${PATH}:/opt/gno/bin" \
             GNOROOT="/opt/gno/src"
 WORKDIR     /opt/gno/src
 FROM        runtime-base AS runtime-tls
-RUN         apt-get update && apt-get install -y expect ca-certificates && update-ca-certificates
+RUN         apt-get update && apt-get install -y expect ca-certificates curl && update-ca-certificates
 
 # slim images
-FROM        runtime-base AS gnoland-slim
+FROM        runtime-tls AS gnoland-slim
+ADD         ./gno.land/genesis /opt/gno/src/gno.land/genesis
+ADD         ./examples /opt/gno/src/examples
+ADD         ./gnovm/stdlibs /opt/gno/src/gnovm/stdlibs
 WORKDIR     /opt/gno/src/gno.land/
 COPY        --from=build /opt/build/build/gnoland /opt/gno/bin/
+RUN         mkdir config
+#TODO: RUN         echo '[rpc]\nladdr = "tcp://0.0.0.0:26657"' > config/config.toml
```

Comment:
Having problems with the config file, really difficult to use. I had to go through the code to exactly know the needed filesystem structure. After that, I only got nil pointer exceptions. It looks like we are expecting to have a complete config file instead of patching some of the specified values on top of the ones provided by flags or defaults...

Reply:
Yes, there is a suggested method to simplify this using #731.

Meanwhile, I recommend checking out these existing docker-compose examples that demonstrate different ways of working with a config file:
-  [docker-integration](https://github.com/gnolang/gno/tree/master/misc/docker-integration)
-  [deployments](https://github.com/gnolang/gno/tree/master/misc/deployments)

Reply:
@ajnavarro 
I'll open up a PR for resolving this config file nonsense so we are unblocked